data_tools/README.md

```diff
@@ -3,4 +3,4 @@ Firebase console > project settings > service accounts
 Add the Firebase Service Account secret as `serviceAccount.json` into the current folder
```

Comment:
I think this is now out of date as we have a path - consider removing.

Reply:
Ah yes fixed!